### PR TITLE
added 2.0 preview blog post

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

File: app/1.0/blog/2016-09-09-polymer-2.0.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Polymer 2.0 is a minimally-breaking change from the current library, with a few very precise goals:
+
+### Take advantage of native [v1 Shadow DOM](https://developers.google.com/web/fundamentals/primers/shadowdom/) and [v1 Custom Elements](https://developers.google.com/web/fundamentals/primers/customelements/) API’s
+
+The primary goal of the Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components.
+
+   Polymer 1.x is built on top of the so-called "v0" Web Components specs, which are supported natively only in Google Chrome; using Polymer in other browsers has always required the use of polyfills.
+
+   Beginning this fall, multiple browsers will be shipping native implementations of the new "v1" specs for Shadow DOM and Custom Elements, yielding better web components performance and reducing the need for polyfills.
```
Comment (addyosmani) [inline app/1.0/blog/2016-09-09-polymer-2.0.md]: May be worth alluding to what browsers we're referring to here for folks that haven't been following along.

**Comment by addyosmani** [comment]
LGTM. If there's time, I might suggest:

- Include a nod to Polymer Summit where folks might learn more about 2.0
- We tell folks to upgrade they may need to make a few changes. As we don't appear to dive into these, may be worth hinting at more details about what changes are needing being documented in the very near future (or something similar).

File: app/1.0/blog/2016-09-09-polymer-2.0.md (hunk 1 above)
Comment (tjsavage) [inline app/1.0/blog/2016-09-09-polymer-2.0.md]: Hm what links to you think might be best to include? I'm thinking https://www.chromestatus.com/features/4667415417847808 and https://developer.apple.com/library/prerelease/content/releasenotes/General/WhatsNewInSafari/Articles/Safari_10_0.html#//apple_ref/doc/uid/TP40014305-CH11-DontLinkElementID_5 but are there good ones for Edge/Firefox?

Comment (addyosmani) [inline app/1.0/blog/2016-09-09-polymer-2.0.md]: Looks like Edge is still under consideration for CE and SD: https://developer.microsoft.com/en-us/microsoft-edge/platform/status/customelements/?q=custom%20elements

I don't have a good one for FF

**Comment by tjsavage** [comment]
Thanks @addyosmani ! Added: a nod to Polymer summit, links to browser announcements, and linked to breaking changes section of README.

**Pull request merged**